ProcessMaker/processmaker
# Implement Importing of Wizard Template

This PR introduces the following functionalities related to Guided Templates:

1. Importing of Wizard Template:

- Upon completion of the helper process, the wizard template is imported.
- If existing assets from the template are present, the user is redirected to handle these assets.
- Upon completion, the user is redirected to the newly created process launchpad.

2. Rerun Wizard Capability:

- Enabled the capability to rerun the wizard helper process from the newly created process launchpad.

3. Nightly Sync of Wizard Templates:

- Enabled the nightly sync of wizard templates.

## How to Test

1. Ensure you have access to a local development environment.
2. Run php artisan processmaker:sync-wizard-templates.
3. Navigate to 'Processes > Guided Templates'.
4. Select a Guided Template.
5. Select 'Use Now'.
6. Complete the process configuration.
7. Upon completion:

-  If there are existing assets from that template, ensure you are redirected to the existing assets page to handle those assets. After completing, ensure redirection to the newly created process launchpad.

- If there are no existing assets, ensure redirection to the newly created process launchpad.

8. Within the process launchpad, ensure the 'Re-run Wizard' link is visible.
9. Select the link.
10. Ensure the helper process is opened in a modal.

## Related Tickets & Packages
Resolves:
- [FOUR-12181](https://processmaker.atlassian.net/browse/FOUR-12181)
- [FOUR-11769](https://processmaker.atlassian.net/browse/FOUR-11769)

ci:next
ci:deploy

## Code Review Checklist


[FOUR-12181]: https://processmaker.atlassian.net/browse/FOUR-12181?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
